packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -66,6 +72,10 @@ const ActivityDetailsSidebar = (props: Props) => {
           gcal
         }
         ...AdhocTeamMultiSelect_viewer
+        organizations {
```

Comment:
+1 this is not updated when a user joins an org, so I assume there is some subscription missing. To test this invite a user with 1 common team so you have 2 in common and then start the meeting without refreshing.